test/db/tools/rz_sign

```diff
@@ -48,8 +48,8 @@ mkdir .tmp
 !rz-sign -o .tmp/hello_world.pat bins/elf/hello_world
 EOF
 EXPECT=<<EOF
-rz-sign: written 11 signatures to .tmp/hello_world.sig.
-rz-sign: written 11 signatures to .tmp/hello_world.pat.
+rz-sign: written 10 signatures to .tmp/hello_world.sig.
```

Comment:
What symbol was removed here?

Reply:
`loc.imp._ITM_deregisterTMCloneTable`, which was pointing at 0 and it was definitely not a function.